public class exception {

    public static void main(String[] args) {
        int a[]={22,233,3};
        try{
            System.out.println("vlaue is: "+a[4]);
        }
        catch(IndexOutOfBoundsException e){
            System.out.println("error occure");
        }
        System.out.println("code run sucesfully........");


        } 
        

       
        
    }
